tests/drivers/dma/mux_req_gen/prj.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_TEST=y
+CONFIG_DMA=y
+CONFIG_DMAMUX_STM32=y
+CONFIG_LOG=y
+CONFIG_DMA_LOG_LEVEL_INF=y
+CONFIG_EXTI_STM32=y
```

Comment:
This should not be required (always 'y')

Reply:
removed